inc/namespace.php

```diff
@@ -219,3 +253,80 @@ function ajax_query_attachments() : void {
 
 	wp_send_json_success( $items->toArray() );
 }
+
+function match_attachments_for_js( array $response, WP_Post $attachment ) : array {
+	global $amf_inserted;
```

Comment:
Instead of a bunch of functions sharing data through a global, can we not create a new class that is responsible for what these functions are doing? If there's value in that, and will try and find some time and create an OOP version of this, given that the current project I'm on also deems uploading to the central site as valuable/required.

Reply:
Looking at it this function and the `$amf_inserted` global can probably be removed entirely now as we have the generic filters to correct the duplicated base URLs. That's all this was doing but the filter can be applied more generally, not sure why I overcomplicated it in the first place like this!